.github/workflows/test.yml

```diff
@@ -14,15 +14,19 @@ jobs:
           - '5.2'
           - '6.0'
           - '6.1'
+          - '7.0'
+        exclude:
+          - { ruby: '2.6.8', rails: '7.0' }
         include:
           # TODO: Move back into matrix when support for Rails 5.2 officially ends
           - { ruby: '3.0.2', rails: '6.0' }
           - { ruby: '3.0.2', rails: '6.1' }
+          - { ruby: '3.0.2', rails: '7.0' }
 
     steps:
       - uses: actions/checkout@v2
       - name: Setup Ruby ${{ matrix.ruby }}
-        uses: actions/setup-ruby@v1
+        uses: ruby/setup-ruby@v1
```

Comment:
`actions/setup-ruby` is deprecated and throw errors that it can't find ruby versions